components/script/dom/trustedtypepolicyfactory.rs

```diff
@@ -85,6 +86,48 @@ impl TrustedTypePolicyFactory {
         // Step 9: Return policy.
         Ok(policy)
     }
+
+    /// <https://w3c.github.io/trusted-types/dist/spec/#abstract-opdef-get-trusted-type-data-for-attribute>
+    fn get_trusted_type_data_for_attribute(
+        element: QualName,
+        attribute: String,
+        attribute_namespace: Option<Namespace>,
+    ) -> Option<DOMString> {
+        // Step 1: Let data be null.
+        let mut data = None;
+        // Step 2: If attributeNs is null, and attribute is the name of an event handler content attribute, then:
+        // TODO(36258): look up event handlers
```

Comment:
It's unclear how to implement this. Relevant spec issue: https://github.com/w3c/trusted-types/issues/520

Reply:
Mmm, we also need this for #36397. We will probably follow gecko's strategy (https://searchfox.org/mozilla-central/rev/6cc05c6620814bf3eb05278ccf3b1f49690b7740/dom/base/nsContentUtils.cpp#4994)